Kritika_Sachdeva/profile/index.html

```diff
@@ -14,14 +14,16 @@
     <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"  crossorigin="anonymous"></script>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"  crossorigin="anonymous"></script>
 
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+
 </head>
 <body>
 
 <!--HEADER-->
 <section id="header">
     <div class="container text-center">
         <div class="user-box">
-            <img src="IMG-4740.JPG">
+            <img src="pp.jpg">
```

Comment:
all static assets must be in a folder named `assets`
For Eg. 

/project
   /assets
      /css
      /scipts
   index.html

Reply:
Done!

Reply:
Made the required changes.